code/modules/admin/smites/retcon.dm

```diff
@@ -0,0 +1,12 @@
+/datum/smite/retcon
+	name = "Retcon"
+	var/timer //how long until the thing attacked by this smite gets deleted
+	var/fade_out_timer //the time it takes to actually do the fade out animation, the victim will always have some time still fully visible
```

Comment:
```suggestion
	/// how long until the thing attacked by this smite gets deleted
	var/timer 
	/// the time it takes to actually do the fade out animation, the victim will always have some time still fully visible
	var/fade_out_timer 
```
Triple-slashed comments above var or proc declarations are picked up by our auto-documenter.
Use `//` for comments or stuff in the middle of a proc, `///` for documenting what a variable is for.